environment.yml

```diff
@@ -2,4 +2,18 @@ name: dotenv
 channels:
   - anaconda
 dependencies:
-  - theano
\ No newline at end of file
+  - python=3.8
```

Comment:
This particular change might make merging portions with master complicated at some point in the future. Would it be possible to copy the old file and rename it `rtd-environment.yml`; then edit `[.readthedocs.yml](https://github.com/bmorris3/dot/blob/master/.readthedocs.yml#L16)` to point to rtd-environment.yml?

Reply:
Updated the changes you requested. I put the `rtd-environment.yml` file inside the `docs/` folder and updated the `.readthedocs.yml` to point to that location.